persistence-cassandra-3.11/src/test/java/io/stargate/db/cassandra/impl/CassandraPersistenceIT.java

```diff
@@ -59,8 +59,6 @@ public static void cleanup() throws IOException {
     //    if (persistence != null) {
     //      persistence.destroy();
     //    }
-
-    FileUtils.deleteDirectory(baseDir);
```

Comment:
Just a hunch I'm exploring for resolving

```
Step #4 - "cassandra-4.0": Caused by: java.io.IOException: Invalid folder descriptor trying to create log replica /tmp/stargate-cassandra-4.0-test1050116150017123924/data/system/table_estimates-176c39cdb93d33a5a2188eb06a56f66e
Step #4 - "cassandra-4.0": 	... 41 common frames omitted
Step #4 - "cassandra-4.0": ERROR [MemtableFlushWriter:1] 2021-03-16 15:12:56,694 LogReplicaSet.java:95 - Failed to create log replica /tmp/stargate-cassandra-4.0-test1050116150017123924/data/system/peers_v2-c4325fbb8e5e3bafbd070f9250ed818e//tmp/stargate-cassandra-4.0-test1050116150017123924/data/system/peers_v2-c4325fbb8e5e3bafbd070f9250ed818e/na_txn_unknowncompactiontype_167df163-866a-11eb-8d41-8fa220fb76af.log
```

Reply:
That's probably because we delete the base dir without shutting down persistence (line 60)... Is `destroy()` still broken? I hope it might have been fixed.

Reply:
I'll uncomment and we can see

Reply:
Still NPE :(

```
Step #4 - "cassandra-3.11": [ERROR] io.stargate.db.cassandra.impl.CassandraPersistenceIT  Time elapsed: 2.156 s  <<< ERROR!
Step #4 - "cassandra-3.11": java.lang.NullPointerException
Step #4 - "cassandra-3.11": 	at io.stargate.db.cassandra.impl.CassandraPersistenceIT.cleanup(CassandraPersistenceIT.java:59)
```

Reply:
Maven truncates the stack trace, I believe the real NPE is in C* code.